[rospy.client][INFO] 2025-06-15 18:38:00,747: init_node, name[/urdf_spawner], pid[9632]
[xmlrpc][INFO] 2025-06-15 18:38:00,748: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2025-06-15 18:38:00,748: Started XML-RPC server [http://192.168.1.124:33093/]
[rospy.init][INFO] 2025-06-15 18:38:00,748: ROS Slave URI: [http://192.168.1.124:33093/]
[rospy.impl.masterslave][INFO] 2025-06-15 18:38:00,748: _ready: http://192.168.1.124:33093/
[rospy.registration][INFO] 2025-06-15 18:38:00,749: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2025-06-15 18:38:00,750: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2025-06-15 18:38:00,848: registered with master
[rospy.rosout][INFO] 2025-06-15 18:38:00,848: initializing /rosout core topic
[rospy.rosout][INFO] 2025-06-15 18:38:00,850: connected to core topic /rosout
[rospy.simtime][INFO] 2025-06-15 18:38:00,851: initializing /clock core topic
[rospy.simtime][INFO] 2025-06-15 18:38:00,852: connected to core topic /clock
[rosout][INFO] 2025-06-15 18:38:00,854: Loading model XML from ros parameter
[rosout][INFO] 2025-06-15 18:38:00,856: Waiting for service /gazebo/spawn_urdf_model
[rospy.internal][INFO] 2025-06-15 18:38:00,894: topic[/clock] adding connection to [http://192.168.1.124:43955/], count 0
[rospy.internal][INFO] 2025-06-15 18:38:00,985: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2025-06-15 18:38:01,158: Calling service /gazebo/spawn_urdf_model
[rosout][INFO] 2025-06-15 18:38:01,645: Spawn status: SpawnModel: Successfully spawned entity
[rospy.core][INFO] 2025-06-15 18:38:01,645: signal_shutdown [atexit]
[rospy.internal][INFO] 2025-06-15 18:38:01,647: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2025-06-15 18:38:01,647: topic[/clock] removing connection to http://192.168.1.124:43955/
[rospy.internal][WARNING] 2025-06-15 18:38:01,647: Unknown error initiating TCP/IP socket to 192.168.1.124:38933 (http://192.168.1.124:43955/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 651, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2025-06-15 18:38:01,655: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 839, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2025-06-15 18:38:01,655: atexit
